syntax = "proto3";

package audioservice;

service AudioService {
  rpc GetAudioFile (AudioRequest) returns (AudioResponse) {}
  rpc ReceiveNewsContent (NewsContentRequest) returns (NewsContentResponse) {}
}

message AudioRequest {
  string news_id = 1;
}

message AudioResponse {
  bytes audio_data = 1;
  string file_name = 2;
}

message NewsContentRequest {
  string news_id = 1;
  string content = 2;
}

message NewsContentResponse {
  bool success = 1;
  string message = 2;
}